tools/pmux/pmux.cpp

```diff
@@ -660,11 +661,15 @@ static int run_cmd(struct pollfd &fd, std::vector<struct pollfd> &fds, char *in,
         for (auto it : active_services) {
             conn_printf(c, "%s\n", it.c_str());
         }
-    } else if (strcmp(cmd, "exit") == 0) {
-        if (c.writable) {
-            return 1;
-        } else {
-            disallowed_write(c, cmd);
+    } else if (strcmp(cmd, "active") == 0) {
+        std::lock_guard<std::mutex> l(active_services_mutex);
+        conn_printf(c, "%d\n", active_services.size());
+        for (auto it : active_services) {
+            conn_printf(c, "%s\n", it.c_str());
+        }
+    } else if (strcmp(cmd, "range") == 0) {
```

Comment:
Update help message to include the new subcommand

Reply:
Yep. Thanks.